examples/freertos/CMakeLists.txt

```diff
@@ -111,6 +129,42 @@ set(INCLUDE_DIRS
     ${CMAKE_CURRENT_SOURCE_DIR}
 )
 
+if(FREERTOS_F100RB_EXAMPLE)
+    set(TARGET_NAME ${PROJ_NAME}-f100)
+    add_executable(${TARGET_NAME})
+
+    target_sources(${TARGET_NAME} PRIVATE ${PROJECT_SOURCES})
+    target_include_directories(${TARGET_NAME} PRIVATE ${INCLUDE_DIRS})
+    target_link_libraries(${TARGET_NAME} PRIVATE
+        ${FREERTOS_F1_NAMESPACE}::Timers
+        ${FREERTOS_F1_NAMESPACE}::Heap::4
+        ${FREERTOS_F1_NAMESPACE}::ARM_CM3
+        HAL::STM32::F1::RCC
+        HAL::STM32::F1::GPIO
+        HAL::STM32::F1::CORTEX
+        CMSIS::STM32::F100RB 
+        STM32::NoSys
+    )
+    if(USE_CMSIS_RTOS)
+        target_link_libraries(${TARGET_NAME} PRIVATE
+            CMSIS::STM32::F1::RTOS
+        )
+    endif()
+    if(USE_CMSIS_RTOS_V2)
```

Comment:
Isn't this an elseif ? (Then adding an else for error if neither RTOS or RTOS_V2 is provided ?

Reply:
Good point.
It was done in the same way as for other boards, e.g. FREERTOS_H743ZI_EXAMPLE

This options validation performed at the beginning:

``` cmake
if(USE_CMSIS_RTOS AND USE_CMSIS_RTOS_V2)
    message(FATAL_ERROR "Can not use USE_CMSIS_RTOS_V2 together with USE_CMSIS_RTOS!")
endif()

if(USE_CMSIS_RTOS)
    message(STATUS "Compiling CMSIS RTOS support")
elseif(USE_CMSIS_RTOS_V2)
    message(STATUS "Compiling CMSIS RTOS V2 support")
endif()
```

Reply:
But thanks to you comment I found typo in my code:
Must be: `CMSIS::STM32::F1::RTOS_V2` instead of `CMSIS::STM32::F1::RTOS_V2`